api-doc/openapi.yaml

```diff
@@ -1,48 +1,514 @@
+      required:
+      - id
+      example:
+        6c28df04-b9d2-4ebe-bd33-58c9b290d929:
+          timestamp: '2021-09-21 15:05:47'
+          name: jsloth
+          message: ota tää :D 🤔🤔
+          amount: 10
+        7c28df04-b9d2-4ebe-bd33-58c9b290d930:
+          timestamp: '2021-09-21 15:05:50'
+          name: luuranki
+          message: 'mee tänne: töihin :))))))))'
+          amount: 5
+    GameData:
+      type: object
+      properties:
+        game:
+          type: string
+        player:
+          type: string
+        start_time:
+          type: string
+        end_time:
+          type: string
+        category:
+          type: string
+        device:
+          type: string
+        published:
+          type: string
+        vod_link:
+          type: string
+        img_filename:
+          type: string
+        player_twitch:
+          type: string
+      required:
+      - game
+      - player
+      - start_time
+      - end_time
+      - category
+      - device
+      - published
+      example:
+        game: Tetris
+        player: jsloth
+        start_time: '2021-09-21 15:05:47'
+        end_time: '2021-09-21 16:05:47'
+        category: any%
+        device: PC
+        published: '1970'
+        vod_link: https://www.youtube.com/watch?v=dQw4w9WgXcQ
+        img_filename: tetris.png
+        player_twitch: jiisloth
+    GameDataAll:
+      type: object
+      properties:
+        id:
+          $ref: '#/components/schemas/GameData'
+      required:
+      - id
+      example:
+        6c28df04-b9d2-4ebe-bd33-58c9b290d929:
+          game: Tetris
+          player: jsloth
+          start_time: '2021-09-21 15:05:47'
+          end_time: '2021-09-21 16:05:47'
+          category: any%
+          device: PC
+          published: '1970'
+          vod_link: https://www.youtube.com/watch?v=dQw4w9WgXcQ
+          img_filename: tetris.png
+          player_twitch: jiisloth
+        6c28df04-b9d2-4ebe-bd33-58c9b290d930:
+          game: Tetris2
+          player: jsloth
+          start_time: '2021-09-21 15:07:47'
+          end_time: '2021-09-21 16:08:47'
+          category: any% no t
+          device: PC
+          published: '1971'
+          vod_link: https://www.youtube.com/watch?v=dQw4w9WgXcQ
+          img_filename: tetris.png
+          player_twitch: jiisloth
+    StreamMetadata:
+      type: object
+      properties:
+        donation_goal:
           type: integer
-        currency:
-          type: string
-          enum:
-            - euro
-      example: {
-        "links": {
-          href: "/target"
-        },
-        "currency": "euro",
-        "target": 1000
-      }
+        current_game_id:
+          type: string
+        donatebar_info:
+          type: array
+          items:
+            type: string
+        counters:
+          type: array
+          items:
+            type: integer
+      required:
+      - donation_goal
+      - current_game_id
```

Comment:
Is there always a game going on? During maintenance break for example?

Reply:
Previously there has been, but it may have had stuff like "Stream starting!" as a game. Removed this as required field